nixos/modules/services/system/cachix-watch-store.nix

```diff
@@ -69,6 +77,7 @@ in
         DynamicUser = true;
         LoadCredential = [
           "cachix-token:${toString cfg.cachixTokenFile}"
+          "signing-key:${toString cfg.signingKeyFile}"
```

Comment:
this should probably be inside a mkIf considering the type can also be null.

Reply:
It should be optional, you are totally right. Should not have marked this ready right before leaving the office yesterday. Thanks!